internal/activity.go

```diff
@@ -247,11 +248,11 @@ func WithActivityTask(
 	tracer opentracing.Tracer,
 ) context.Context {
 	var deadline time.Time
-	scheduled := time.Unix(0, task.GetScheduledTimestamp())
```

Comment:
How do we handle this change in behavior? If `ScheduledTime` is set to `0` value; previously behavior would make `scheduled` equal `time.Unix(0,0)` but it is now `&time.Time{}`? Is this a problem?

Reply:
In this particular case it is not a problem at all because these values can't be zero.